WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -202,9 +202,10 @@ class PluginViewModel: Observable {
     }
 
     private func autoUpdatesRow(plugin: Plugin?, capabilities: SitePluginCapabilities?) -> ImmuTableRow? {
+        // Note: All plugins on atomic sites are autoupdated, so we do not want to show the switch
         guard let autoUpdatePlugin = plugin,
             let siteCapabilities = capabilities,
-            siteCapabilities.autoupdate && !autoUpdatePlugin.state.automanaged else { return nil }
+            !isAutomatedTransfer(site: site) && siteCapabilities.autoupdate && !autoUpdatePlugin.state.automanaged else { return nil }
```

Comment:
Just the tiniest style nitpick (and a retroactive nit to myself!): I'd rewrite that as:

```swift
 guard let autoUpdatePlugin = plugin,
            let siteCapabilities = capabilities,
            siteCapabilities.autoupdate,
            !isAutomatedTransfer(site: site),
            !autoUpdatePlugin.state.automanaged else { return nil }
```

to make it obvious that all of the conditions have to be met here. Dunno why I went with `&&` there earlier :)